fun exitFailure(): Void {
    py#exit(1)
}


fun exitSuccess(): Void {
    py#exit(0)
}


fun exit(status: Number): Void {
    py#exit(status.data)
}
